src/Client.php

```diff
@@ -280,20 +280,20 @@ class Client implements ClientInterface {
     'schedule' => [
       'get' => [
         'action' => 'GET',
-        'uri'    => '/apis/extension.shepherd/v1/namespaces/{namespace}/schedules/{name}',
+        'uri'    => '/apis/extension.shepherd/v1/namespaces/{namespace}/backupscheduleds/{name}',
```

Comment:
Err bit awkward naming. Shouldn't it be 
scheduledbackups or
backupschedules ?

Reply:
Yeah it's what kubebuilder spat out, I'll see if @nicksantamaria can get it changed. We'd need to do it before the crds are installed though.

Reply:
I went back and forth a bit on the naming - landed on `BackupScheduled` for 2 main reasons.

1. Code organisation in `shepherd-operator` is a bit nicer, with backup related files all grouped together alphabetically.
2. It keeps the CRDs grouped together alphabetically when running `oc get` and `oc get crd`

Unfortunately the plural naming is a bit awkward, and we've hit this with other operators too. I prefer to just use the singular.